drivers/NFSSR.py

```diff
@@ -92,6 +92,10 @@ def load(self, sr_uuid):
         if self.dconf.has_key('useUDP') and self.dconf['useUDP'] == 'true':
             self.transport = "udp"
         self.nfsversion = nfs.validate_nfsversion(self.dconf.get('nfsversion'))
+        if self.dconf.has_key('options'):
```

Comment:
Not an error and I do understand it is to adhere to code above but
has_key() is deprecated.

Sooner or later we should start "breaking style compatibility" and write standard compliant code.

Up to you